modules/shared/system_module_version.mk

```diff
@@ -34,5 +34,14 @@ endif
 # Bump by 1 for every updated bootloader image for a release with the same v0.x.* base.
 BOOTLOADER_VERSION ?= 301
 
-# the version of the bootloader that the system firmware requires
+# The version of the bootloader that the system firmware requires
+# NOTE: this will force the device into safe mode until this dependency is met, which is why
+# this version usually lags behind the current bootloader version, to avoid non-mandatory updates.
+# gen 2 devices
+ifneq (,$(filter $(PLATFORM_ID),6 8 10))
 BOOTLOADER_DEPENDENCY = 201
+endif
+# gen 3 devices
+ifneq (,$(filter $(PLATFORM_ID),12 13 14 22 23 24))
```

Comment:
I'm a little uneasy about explicitly listing Gen3 platforms...I feel we should apply the latest bootloader unless otherwise specified? This is the kind of thing that is easy to forget to update.

Reply:
Updating the bootloader is kind of a risk vs reward proposition.  Thus the reason why the dependency is currently lagging the actual version.  When we get to the default release though, that seems like a good time to take the minor step to sync everything up as I stated in the PR.  I'm not sure what the issue is with explicitly listing the platforms, we have 9 modular platforms and they are all accounted for.